config/schema/document_types/vehicle_recalls_and_faults_alert.json

```diff
@@ -0,0 +1,111 @@
+{
+  "fields": [
+    "fault_type",
+    "alert_issue_date",
+    "item_type",
+    "manufacturer",
+    "item_model",
+    "serial_number",
+    "build_start_date",
+    "build_end_date"
+  ],
+
+  "allowed_values": {
+    "fault_type": [
+      {
+        "label": "Recall",
+        "value": "recall"
+      },
+      {
+        "label": "Non-urgent fault",
+        "value": "non_urgent_fault"
+      }
+    ],
+
+    "item_type": [
+      {
+        "label": "Vehicle",
+        "value": "vehicle"
+      },
+      {
+        "label": "Baby seat",
+        "value": "baby-seat"
+      },
+      {
+        "label": "Tyres",
+        "value": "tyres"
+      },
+      {
+        "label": "Parts",
+        "value": "parts"
+      },
+      {
+        "label": "Agricultural equipment",
+        "value": "agricultural-equipment"
+      },
+      {
+        "label": "Other accessories",
```

Comment:
Changing the schema to add allowed values isn't very difficult, but does require a deploy so is a bit fiddly.

Based on your comments, this will be fine for now.